core/src/test/java/org/jclouds/internal/BaseViewTest.java

```diff
@@ -104,7 +104,7 @@ public void testPeanutButterDidntTurnIntoWine() {
          wine.unwrap(typeToken(PeanutButter.class));
          fail();
       } catch (IllegalArgumentException e) {
-         assertEquals(e.getMessage(), "backend type: org.jclouds.internal.BaseViewTest$Water not assignable to org.jclouds.internal.BaseViewTest$PeanutButter");
+         assertEquals(e.getMessage(), "backend type: org.jclouds.internal.BaseViewTest$Water is not a supertype of org.jclouds.internal.BaseViewTest$PeanutButter");
```

Comment:
Does this message depend on the Guava version being used? In that case we need to change the assertion, otherwise our builds that check different versions of Guava will fail.